pom.xml

```diff
@@ -789,6 +790,22 @@
         <type>test-jar</type>
       </dependency>
 
+      <dependency>
+        <groupId>org.apache.arrow</groupId>
+        <artifactId>arrow-vector</artifactId>
+        <version>${arrow.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.apache.arrow</groupId>
+        <artifactId>arrow-memory-netty</artifactId>
```

Comment:
Is this used?

Reply:
both `arrow-memory-netty` and `arrow-memory-unsafe` are internal implementation.
Did some internal benchmark and found out `arrow-memory-netty` is more performed.